### PR TITLE
Handle NaN results from Presto server

### PR DESCRIPTION
**Pull request by nezihyigitbasi**
This change depends on PR #34

### CODE REVIEW

File: driver/prestoresults.d [hunk 1]
```diff
@@ -26,10 +26,17 @@ import presto.client.queryresults : ColumnMetadata;
...
     final switch (columnData.type) {
         case JSON_TYPE.STRING:
-            result.addNextValue(Variant(columnData.str));
+            if (resultDataType == "string") {
+                result.addNextValue(Variant(columnData.str));
+            } else {
+                //for some scalar functions that return double, we may get a "NaN" back
+                if (columnData.str == "NaN") { 
+                    result.addNextValue(Variant(double.nan));
+                }
```
Comment (markisaa) [inline driver/prestoresults.d:38]: Should we throw an exception in the `else` case?

**Comment by markisaa** [comment]
Awesome. Soon as the comment is tended to this is ready to merge :)

**Comment by nezihyigitbasi** [comment]
Somehow I don't see my recent pushes in this PR, so closing this one & opened a new PR (#37) for this change.

**Comment by markisaa** [comment]
Did you try doing a:
git co master
git pull --rebase upstream master 
git co NaN_handling
git rebase master 
git push origin NaN_handling -f

That may be useful for such cases in the future.

**Comment by nezihyigitbasi** [comment]
yep, I rebased the PR and pushed it, but this PR didn't get updated somehow.

**Pull request closed without merging**